core/modules/dblog/dblog.module

```diff
@@ -152,9 +152,13 @@ function dblog_watchdog(array $log_entry) {
  */
 function dblog_form_system_logging_settings_alter(&$form, $form_state) {
   $site_config = config('system.core');
-  $form['dblog_row_limit'] = array(
+  $form['dblog'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Database log'),
+  );
+  $form['dblog']['dblog_row_limit'] = array(
     '#type' => 'select',
-    '#title' => t('Database log messages to keep'),
+    '#title' => t('Messages to keep'),
     '#default_value' => $site_config->get('log_row_limit'),
     '#options' => array(0 => t('All')) + backdrop_map_assoc(array(100, 1000, 10000, 100000, 1000000)),
     '#description' => t('The maximum number of messages to keep in the database log. Requires a <a href="@cron">cron maintenance task</a>.', array('@cron' => url('admin/reports/status')))
```

Comment:
While at it, can we please chance the link here to point to `admin/config/system/cron` instead of the status report page?

```suggestion
    '#description' => t('The maximum number of messages to keep in the database log. Requires a <a href="@cron">cron maintenance task</a>.', array('@cron' => url('admin/config/system/cron')))
```